lib/features/public_asset/presentation/public_asset_controller.dart

```diff
@@ -206,8 +224,13 @@ class PublicAssetController extends BaseController {
     if (session == null || accountId == null) return;
 
     final fileInfo = platformFile.toFileInfo();
-    final uploadUri = session!.getUploadUri(accountId!, jmapUrl: dynamicUrlInterceptors.jmapUrl);
-    consumeState(_uploadAttachmentInteractor.execute(fileInfo, uploadUri));
+    try {
+      final uploadUri = session!.getUploadUri(accountId!, jmapUrl: dynamicUrlInterceptors.jmapUrl);
+      consumeState(_uploadAttachmentInteractor.execute(fileInfo, uploadUri));
+    } catch (e) {
+      log('PublicAssetController::_uploadFileToBlobAction: $e');
+      consumeState(Stream.value(Left(UploadAttachmentFailure(e, fileInfo))));
```

Comment:
idem

Reply:
Agree with it, but make sure all the handle for `UploadAttachmentFailure` still work well